knative-operator/cmd/manager/main.go

```diff
@@ -118,7 +118,7 @@ func main() {
 	hookServer.Register("/mutate-knativekafkas", &webhook.Admission{Handler: knativekafka.NewConfigurator(decoder)})
 	hookServer.Register("/validate-knativekafkas", &webhook.Admission{Handler: knativekafka.NewValidator(mgr.GetClient(), decoder)})
 
-	if err := setupMonitoring(cfg); err != nil {
+	if err := setupServerlesOperatorMonitoring(cfg); err != nil {
```

Comment:
```suggestion
	if err := setupServerlessOperatorMonitoring(cfg); err != nil {
```

and the corresponding definition

Reply:
Why is "ServerlessOperator" needed here as extra context information ? Is it not clear that this is about the setting up monitoring for Knative/Serverless ?

Reply:
I added this for clarity because there are other funcs with similar functionality.